src/faostat_data_primap/helper/definitions.py

```diff
@@ -422,3 +422,17 @@
         "institution": ("Food and Agriculture Organization of the United Nations"),
     },
 }
+
+# TODO maybe this should live somewhere else?
+# Definition of the domains and releases to be read
+domains_and_releases_to_read = {
+    "2024": (
+        # ("farm_gate_agriculture_energy", "2024-11-14"),
+        # ("farm_gate_emissions_crops", "2024-11-14"),
+        # ("farm_gate_livestock", "2024-11-14"),
+        # ("land_use_drained_organic_soils", "2024-11-14"),
+        # ("land_use_fires", "2024-11-14"),
+        # ("land_use_forests", "2024-11-14"),
+        ("pre_post_agricultural_production", "2024-11-14"),
+    )
```

Comment:
This is the configuration for the data reading step. That way we can choose which domain and release to read.